lambda_function.py

```
@@ -186,16 +240,23 @@ def lambda_handler(event, context):
 
         # Convert coordinates and prepare output
         output = StringIO()
-        output.write("timestamp longitude latitude elevation\n")
+        output.write("timestamp longitude latitude elevation roll pitch yaw\n")
 
-        for timestamp, x, y, z in trajectories:
+        for timestamp, x, y, z, qx, qy, qz, qw in trajectories:
             try:
                 lon, lat, ele = convert_coordinates(
                     x, y, z, first_lat, first_lon, first_ele, rotation_angle, scale
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)